scylla/tests/integration/authenticate.rs

```diff
@@ -64,7 +64,7 @@ impl AuthenticatorProvider for CustomAuthenticatorProvider {
 #[ignore]
 async fn custom_authentication() {
     setup_tracing();
-    let uri = std::env::var("SCYLLA_URI").unwrap_or_else(|_| "127.0.0.1:9042".to_string());
+    let uri = std::env::var("SCYLLA_URI").unwrap_or_else(|_| "172.43.0.2:9042".to_string());
```

Comment:
:thinking: I don't think it's a good idea to adjust the code to our CI. Let's instead add the env override to the workflow (see my other comment).

Reply:
fixed